snow/consensus/snowstorm/conflicts/conflicts.go

```diff
@@ -30,35 +30,35 @@ type transitionNode struct {
 
 // Conflicts implements the [snowstorm.Conflicts] interface
 type Conflicts struct {
-	// track the currently processing txs. This includes
-	// transactions that have been added to the acceptable and rejectable
-	// queues.
-	// Transaction ID --> The transaction with that ID
+	// tracks the currently processing txs. This includes
+	// transactions in [acceptable] and [rejectable].
+	// Transaction ID --> The transaction with that ID.
 	txs map[ids.ID]Tx
 
 	// tracks the currently processing transitions
+	// Transition ID --> Data about that transition.
 	transitionNodes map[ids.ID]transitionNode
 
-	// track which txs are currently consuming which utxos.
-	// UTXO ID --> IDs of transactions that consume the UTXO
+	// tracks which txs consume which utxos.
+	// UTXO ID --> IDs of transactions that consume the UTXO.
 	utxos map[ids.ID]ids.Set
 
 	// conditionallyAccepted tracks the set of txIDs that have been
 	// conditionally accepted, but not returned as fully accepted.
+	// A transaction is conditionally accepted if it has received enough
+	// consecutive successful polls but has unmet dependencies.
 	conditionallyAccepted ids.Set
 
-	// accepted tracks the set of transition IDs that have been added to the acceptable
-	// queue.
-	accepted ids.Set
+	// acceptableIDs is the set of IDs of transactions in [acceptable].
```

Comment:
This is actually the set of transitionIDs that have been accepted in some transaction in acceptable.